src/main/kotlin/org/rust/ide/refactoring/inlineFunction/RsInlineFunctionProcessor.kt

```diff
@@ -226,6 +230,51 @@ class RsInlineFunctionProcessor(
         }
     }
 
+    private fun removeFunctionUseStatements(caller: PsiElement, reference: RsFunction) {
+        usagesAsReference.filter {
+            it.element.ancestorOrSelf<RsUseSpeck>() != null
+        }.forEach {
+            val useGroup = getLowestGroupWithMultipleChildren(it)
+            val useGroupChildrenSize = useGroup?.useSpeckList?.size
+
+            if (useGroupChildrenSize != null && useGroupChildrenSize > 1) {
+                val useSpeck = it.element.ancestorOrSelf<RsUseSpeck>()
+                val extendedSpeck = getExtendedUseSpeck(useGroup, useSpeck)
+                val speckIndexInGroup = useGroup.children.indexOf(extendedSpeck)
+                val childrenWithLeaves = useGroup.childrenWithLeaves
+                val speckIndexInGroupIncludingLeaves = childrenWithLeaves.indexOf(extendedSpeck)
+
+                // Remove comma if needed
+                if (speckIndexInGroup != 0) {
+                    childrenWithLeaves.take(speckIndexInGroupIncludingLeaves).last { groupChild -> groupChild.text == "," }.delete()
+                } else {
+                    childrenWithLeaves.drop(speckIndexInGroupIncludingLeaves).first { groupChild -> groupChild.text == "," }.delete()
+                }
+
+                extendedSpeck?.delete()
```

Comment:
I think that you can use `extendedSpeck?.deleteWithSurroundingComma()` instead.

Reply:
Totally works, didn't know about this method. Cleaned up accrodingly 😄 .